contracts/IdentityManager.sol

```diff
@@ -208,4 +208,12 @@ contract IdentityManager {
         t := eq(a, and(mask, calldataload(4)))
     }
   }
+
+  function isOwner(address identity, address user) constant returns (bool) {
```

Comment:
yeah, maybe change to `owner` and `recoveryKey` in the `isRecovery` function. Then it is consistant with everywhere else.